apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -73,6 +73,43 @@ def extract_dag(
     return depends_graph, group_belonging
 
 
+def randomize_distractors_in_ordered_blocks(all_blocks: list[OrderBlocksAnswerData]) -> None:
```

Comment:
rename function to just `randomize_distractors` ? to go along with the change to the attribute name